src/components/checks/[share_token]/practice/PracticeQuestionNavigation.tsx

```
@@ -1,17 +1,28 @@
+        <button
+          aria-label={`select question ${i}`}
+          type='button'
+          data-selected={currentQuestionIndex === i}
+          disabled={currentQuestionIndex === i}
+          key={q.id}
+          className={cn('group relative w-fit enabled:hover:cursor-pointer disabled:hover:cursor-auto')}
+          onClick={() => navigateToQuestion(i)}>
+          <span className='absolute right-0 bottom-2 left-0 text-center text-sm text-neutral-300 group-enabled:group-hover:font-semibold group-data-[selected="true"]:font-semibold'>{i + 1}</span>
+          <div
+            className='h-2 w-16 rounded-xl bg-neutral-600 shadow-neutral-500 group-enabled:group-hover:bg-neutral-500 group-data-[selected="true"]:bg-neutral-400 group-data-[selected="true"]:shadow-sm dark:group-active:bg-neutral-500/60'
+            children=''
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)